dev/tslint.json

```diff
@@ -57,6 +57,8 @@
                 "keyIndex": 0,
                 "messageIndex": 1
             }
-        ]
+        ],
+        "no-implicit-dependencies": false,
```

Comment:
I actually really like this rule. We had a situation in the app service extension where `azure-storage` was listed as a dev dependency on accident which resulted in an unusable vsix (we didn't release the vsix - but we did send it to people to test haha). This rule would've caught that. Also you can exclude specific modules like this:
```
"no-implicit-dependencies": [
    true,
    [
        "vscode"
    ]
],
```